src/pybind/ceph_argparse.py

```diff
@@ -492,11 +492,16 @@ class CephFilepath(CephArgtype):
     Openable file
     """
     def valid(self, s, partial=False):
+        cleanup = False
```

Comment:
ahh, right. probably we should be checking if the directory is writable. what do you think?